projects/views.py

```diff
@@ -401,6 +419,30 @@ def view_project(request, project_id):
                     "An unexpected error occurred while attempting "
                     "to change role for this user. Please try again",
                 )
+        elif "su_budget_user" in request.POST:
+            budget_user = User.objects.get(username=request.POST["user_ref"])
+            set_budget_for_user_in_project(
+                budget_user, portal_project, request.POST["su_budget_user"]
+            )
+            messages.success(
+                request,
+                (
+                    f"SU budget for user {budget_user.username} "
+                    f"is currently set to {request.POST['su_budget_user']}"
+                )
+            )
+        elif "default_su_budget" in request.POST:
+            portal_project.default_su_budget = request.POST["default_su_budget"]
+            portal_project.save()
+            for u in users:
```

Comment:
I think for consistency, this should be removed to the balance service check. That way, all of the chargebudget objects are created in the same place, when the user first tries to use SUs.